README_KOR.md

```diff
@@ -0,0 +1,197 @@
+  - [Build configuration](#build-configuration)
+- [How to play](#how-to-play)
+
+
+## History 
+
+시초가 되는 *Mario War*는 2002년 *Samuele Poletto* 가 만들었는데, 4명의 슈퍼마리오 캐릭터가 여러 레벨에서 서로의 머리에 점프를 함으로써 대결을 할 수 있었다. 이 게임은 파스칼/어셈블리어(Pascal/Assembly)로 제작이 되었고, DOS용으로 배포 되었다. 이후 버전들에는 맵 에디터도 포함되었다.
+
+2004년에 *72dpiarmy*라는 포럼의 설립자 *Florian Hufsky*가 오픈소스로 새롭게 작성하면서 *Super Mario War*로 시작됐다. 이 버전에서는 커스텀 캐릭터와, 추가적인 게임모드, 메카닉 맵과, 일반 아이템과 강화 아이템들까지 소개 했었다. 유저가 만든 커스텀 컨텐츠들은 `smwstuff.com` 라는 수천개의 맵과 스킨들을 이용할 수 있는 사이트에 저장되었다(지금은 사라진 사이트이다). 오픈 소스라는 특성으로 인해 이 게임은 거의 모든 데스크탑과 콘솔 시스템으로 이식되었다.
+
+2009년 말, Florian이 죽고 게임의 개발은 점차 느려졌고 결국 Super Mario War 1.8 베타 2 가 마지막 공식 릴리즈 버전으로 개발이 중단 되었다. 개발을 계속하거나 다시 시작하려는 여러 시도들이 있었지만, 그 시도들 모두 장기적으로 봤을 때 성공한 것 같지 않았다. 또한 기술적인 문제로 인해 `smwstuff.com` 사이트 전체와 모든 내용이 손실 되었고, 포럼에서 이용가능 하던 업로더의 백업들의 부분도 같이 손실 되었다.
+
+이것이 2014년쯤 부터 내가 포크를 하여 작업을 시작한 이유이며, 크로스 플랫폼을 구현하여 서로 다른 플랫폼에서 네트워크 멀티플레이를 구현하는 것이 초기 목표이다. 또한 포럼에서 흩어진 콘텐츠들을 한 곳에 수집할 사이트를 다시 한 번 만들고 싶었다. 주소 : [`smwstuff.net`](http://smwstuff.net).
+
+게임의 경우, 불행하게도 원본 소스코드의 수준과 구조가 효과적인 작업이 불가능하다는 것이 확실해졌다. 대부분의 소스 파일에는 약 10000줄가량이었고 1000개 이상의 LOC함수가 공통적이며, 대부분의 전역 변수를 수정 했고, 100개의 분기를 가진 `if-else`도 있었다.
+
+결국 나는 네트워크 멀티플레이를 작동하게 할 수는 있었지만, 완벽하지는 않았다. 좋은 조건에서는 로컬 네트워크에서는 잘 작동했지만, 미묘한 버그와 지연이 일반적으로 인터넷을 이용한 게임플레이를 불안하게 했다. 적절한 구현을 위해서는 게임의 몇가지 핵심 부분과 코드를 유지 관리하기 위해 수백개의 정리와 리팩터링 패치를 다시 디자인해야 할 것입니다. 물론 이런 종류의 프로젝트를 통해 언제든지 어떤 회사에서든 중단하라는 연락을 받을 수 있다.
+
+2016년은 나에게 이 프로젝트에 시간을 더 이상 투자하지 않은 시점이었다. 장기적으로 보면 전체를 새로 쓰는 것은 원래 게임을 패치하는 것보다 더 빠르고 효과적일 수 있었다. 어느 쪽이든, 나는 프로젝트를 이전보다 더 나은 상태로 두고 미래 개발자에게 유용하기를 바란다. 재밌게 즐기길!
+
+
+## About
+
+Super Mario War는 Super Mario의 멀티플레이 버전 게임이다. 게임의 목표는 마리오가 가능한 많이 승리하는 것이다. 게임에는 서로 다른 여러개의 게임 모드가 있는데, 깃발잡기, 일인자, 데스매치, 팀 데스매치, 토너먼트 모드, 코인 모으기, 경주 그외에 여러가지가 있다. 이 게임에는 레벨 에디터가 포함되어 있어 맵을 처음부터 새롭게 만들거나 좋아하는 마리오 게임에서 구역을 재창조 할 수 있습니다. 당신의 상상력이 한계를 결정합니다! 최근 버전에는 토너먼트와 통합된 슈퍼마리오 브라더스 3 와 같은 플레이를 만들기 위해서 지정된 조건으로 여러 레벨을 묶을 수 있는 월드 에디터도 포함되어 있다. 이 게임은 Nintendo와 Samuele Poletto가 만든 원조 팬메이드 게임 Mario War에 대한 찬사다.
+
+이 게임은 Nintendo의 아트워크와 사운드를 사용한다. 우리는 이 비상업적 팬 게임이 공정한 사용으로 자격을 얻기를 바란다. 단지 우리는 Nintendo의 캐릭터와 게임을 우리가 얼마나 좋아하는지 보여주기 위해 이 게임을 만들고 싶었다.
+
+### Features
+
+- 최대 4명의 플레이어가 즐길 수 있는 데스매치
+- 다양한 게임 모드 (featuring GetTheChicken, Domination, CTF, ...)
+- online and local multiplayer온라인과 로컬 멀티플레이
+- Comes with the leveleditor레벨 에디터와 함께 제공 - 자신만의 맵 제작...
+- ... 그리고 많은 사람들의 그들만의 맵 제작. 이로 인해 현재 맵은 1000개를 넘어간다
+- 막대기로 원숭이를 찌르는 것들보다 훨씬 재밌다(텀블링 몽키?)
+- 모든 게임에 관련된 소스코드는 무료로 이용가능하
+- SDL을 사용하여 windows, linux, mac 등에 모두 호환된다
+- CPU 플레이어와 함께 할 수 있다
+- 당신을 즐겁게 할 것이다!!!
+
+### Supported platforms
+
+- Linux
+- Windows
+- Mac OS X
+- ARM devices (예를 들어 Raspberry Pi-라즈베리 파이)
+- XBox (?)
+- Android (실험중)
+- asm.js (실험중)
+
+
+## Building instructions
+
+### Requirements
+
+- C++11를 지원하는 컴파일러 (예 gcc-4.8)
+- CMake (>= 2.6)
+- SDL (1.2 or 2.0) 그리고,
+    - SDL_image
+    - SDL_mixer
+- zlib
+- yaml-cpp (포함됨)
+- ENet (선택, 포함됨)
+
+저런 조건을 위해 패키지 매니저를 사용 할 수 있다
+각 시스템 별 코드:
```

Comment:
Why was this line broken into two?

Reply:
In Korean, that line is contextually split into two lines.

Reply:
I *think* GitHub will still display it as one line if they aren't separated by a blank line, just asking because of that.

Reply:
Oh, thanks good advice. I'll separate them grammatically.